lib/smart_app_launch/client_suite/token_request_bsca_verification_test.rb

```diff
@@ -17,7 +17,16 @@ class SMARTClientTokenRequestBackendServicesConfidentialAsymmetricVerification <
     title 'Verify SMART Token Requests'
     description %(
         Check that SMART token requests are conformant.
-      )
+    )
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@229',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@230',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@233',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@234',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@235',
```

Comment:
@235 seems like a better fit for lib/smart_app_launch/client_suite/authorization_request_verification_test.rb. Please check and move/remove if you agree.

@235
[When requesting] an access token via HTTP POST to the FHIR authorization server’s token endpoint URL [the] `scope` [parameter is] `required` [and SHALL contain] … the scope of access requested ... following the [SMART Scopes syntax](https://hl7.org/fhir/smart-app-launch/STU2.2/scopes-and-launch-context.html)

Reply:
Removed in [9b4d4ad](https://github.com/inferno-framework/smart-app-launch-test-kit/pull/105/commits/9b4d4adf0a9945b994d01c6ee21808ba05be91a6).